ansible/roles/boom-waffle-config/tasks/main.yml

```diff
@@ -1,5 +1,14 @@
 ---
 
-- name: Disable power-saving mode on wlan0
+- name: Disable power-saving mode on wlan0 / wlan1 at boot
   become: yes
-  shell: iw dev wlan0 set power_save off
+  blockinfile:
+    dest: '/etc/rc.local'
+    insertafter: '\/bin\/'  # after /bin/sh or /bin/bash line
+    block: |
+      iw dev wlan0 set power_save off || true
+      iw dev wlan1 set power_save off || true
+
+- name: Disable power-saving mode on wlan0 / wlan1 now
+  become: yes
+  shell: iw dev wlan0 set power_save off || true; iw dev wlan1 set power_save off || true
```

Comment:
Yea I could probably do both of these "with items" for DRYer code, maybe in next update.

Reply:
This won't be idempotent.  That's once thing about writing ansible, you want each command to be idempotent so ansible works correctly.  You should always be able to run the playbook again and it should show that there were no changes necessary.  I can show you how to make something like this idempotent later